.github/workflows/publish_hove_images_aws.yml

```diff
@@ -35,13 +34,13 @@ jobs:
           then
             echo "Workflow triggered due to finished upstream workflow"
             echo "::set-output name=branch::${{ github.event.workflow_run.head_branch }}"
-            if [[ '${{ github.event.workflow_run.head_branch }}' == 'dev' ]]
+            if [[ '${{ github.event.workflow_run.head_branch }}' == 'release' || '${{ github.event.workflow_run.head_branch }}' =~ v[0-9]+.[0-9]+.[0-9]+ ]]
             then
-              echo "::set-output name=tag::dev"
-            else :
-              echo "Realease branch, collecting last version"
+              echo "Release branch, collecting last version"
               VERSION=$(curl https://api.github.com/repos/hove-io/navitia/tags | jq --raw-output '.[0].name')
               echo "::set-output name=tag::$VERSION"
+            else :
```

Comment:
Should you also test that you're on dev here?
To avoid pushing something else, not sure.

Reply:
I would say that's a feature more than a risk. Someone with a custom branch that would be able to trigger this part of code, might actually want this to be published in order to test... :shrug: Let's give it a chance :smile: